topics/icrc_34_delegation.md

```diff
@@ -54,34 +64,38 @@ trust the relying party following the [ICRC-28](./icrc_28_trusted_origins.md) st
   "params": {
     "scopes": [
       {
-        "method": "icrc34_get_global_delegation",
-        "targets": [
-          "xhy27-fqaaa-aaaao-a2hlq-cai"
-        ]
+        "method": "icrc34_delegation"
       }
     ]
   }
 }
 ```
 
-## Request
+## icrc25_supported_standards
 
-**`principal` (`text`):** Principal that the global delegation is requested for.  
-**`publicKey` (`blob`):** Public key that receives the global delegation as described in
-the [IC interface specification, signatures section](https://internetcomputer.org/docs/current/references/ic-interface-spec/#signatures).  
-**`targets` (`text` array):** A list of target canister ids (textual representation) the scope is restricted to.  
-**`maxTimeToLive` (`text` array):** (Optional) Expiration of the delegation in nanoseconds, signer can still choose to
+An ICRC-25 compliant signer must implement the [icrc25_supported_standards](icrc_25_signer_interaction_standard.md#icrc25_supported_standards) method which returns the list of supported standards. 
+Any signer implementing ICRC-34 must include a record with the name field equal to "ICRC-34" in that list.
+
+## Request params
+
+- `publicKey` (`text`): A DER encoded public key of a supported signature scheme, as described [here](https://internetcomputer.org/docs/current/references/ic-interface-spec/#signatures).
+- `targets` (`text` `array` optional): A list of target canister ids (textual representation) Account Delegations can execute transactions
+for on behalf of the user.
+- `maxTimeToLive` (`text` optional): Expiration of the delegation in nanoseconds, though signer can still choose to
 return a delegation with a shorter expiration.
 
-### Example RPC Request
+### Example RPC Delegation Request
+
+Requests for a delegation can optionally include `targets`, enabling signers to verify the security of returning an Account 
+Delegation (via ICRC-28). If the array of `targets` is empty, signers **SHOULD** only offer users the option to connect 
```

Comment:
My intention was to make it clearer that this is _not_ the case where targets is `undefined`.

```suggestion
Delegation (via ICRC-28). If the array of `targets` is set to an empty array, signers **SHOULD** only offer users the option to connect 
```